include/jemalloc/internal/hpa_opts.h

```diff
@@ -59,6 +104,43 @@ struct hpa_shard_opts_s {
 	 * Maximum number of hugepages to purge on each purging attempt.
 	 */
 	ssize_t experimental_max_purge_nhp;
+
+	/*
+	 * When the number of inactive bytes in a hugepage is >=
+	 * purge_threshold, the page is purgable.  Setting this to HUGEPAGE
+	 * would only purge empty pages if using hugify_style_eager or
```

Comment:
What is `HUGEPAGE` in this context?  Do you mean "setting this to HUGEPAGE bytes" or the string "HUGEPAGE"?